.travis/custom.sh

```diff
@@ -23,5 +23,5 @@ shift 2
 
 # Write your custom commands here that should be run when `tox -e custom`:
 if [[ -z "${TRAVIS}" ]] || lsr_check_python_version ${ENVPYTHON} -eq '3.6'; then
-  (set -x; cd ${TOPDIR}/tests && ${ENVPYTHON} ./ensure_non_running_provider.py)
+  (set -x; cd ${TOPDIR}/tests; ${ENVPYTHON} ./ensure_non_running_provider.py)
```

Comment:
why this change? you don't care whether cd has succeeded or failed?

Reply:
`set -e` should handle this (exit immediately if a command exits with a non-zero status).